recipes/xrft/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "xrft" %}
+{% set version = "0.1.dev" %}
+{% set sha256 = "b7235736b0188feeb8d25d3368ac5f7eb0d7d38129080687e82ee906b587b69a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/rabernat/{{ name }}/archive/master.tar.gz
```

Comment:
So this isn't really a tag and it is just building from `master`. 😕 Can you ask upstream to add a tag and use that instead?